docs/running-on-coclobas.md

```diff
@@ -163,5 +163,39 @@ If you'd like to submit with your own data, you can inspect the options of
 You can watch the run execute through the Ketrew Web UI found at `./disco.sh
 ketrew-ui`; the result of the run can be found in `$BIOKEPI_WORK_DIR/results`.
 
+### Adding more NFS servers to make simultaneous Epidisco runs efficient
 
+If you are planning to run more than 2-3 epidisco workflows on the same cluster,
+the default single-node NFS server can easily become a bottleneck for all jobs,
+where each epidisco workflow will end up competing for the NFS resources and
+this will dramatically slow all workflows down.
 
+To overcome this issue, you can add more NFS servers to the Epidisco setup
+and let each workflow use a separate NFS as a working directory. The following
+command, for example, will create a new NFS server with 1TB storage and mount
+it to `/nfs-pool2/`:
```

Comment:
done (mentioned this right above the `add-new-nfs` command example as a comment.